packages/Form/Input/date/src/datepicker.scss

```diff
@@ -84,7 +84,7 @@ $border-radius-width: 3px;
   }
 
   &__input-container {
-    z-index: 2;
+    z-index: 3;
```

Comment:
Instead of changing this z-index, just leave it "as-is" and remove z-index on .glyphicon-calendar (line 17). Should do the trick.

Reply:
I test it asap, thanks for the review @gcruchon and @arnaudforaison. It is quite interesting !

Reply:
(In //, I would like the thoughts of @guillaumechervet  @guillaumechervetaxa  @guillaume-chervet and @samuel-gomez  @samuel-gomez-axa)

Reply:
indeed, this solution may not work all the time, you have to test in many cases. but I don't think using z-indexes is bad practice.